lib/onc_certification_g10_test_kit/multi_patient_api_stu2.rb

```diff
@@ -0,0 +1,54 @@
+require_relative 'bulk_data_authorization'
+require_relative 'bulk_data_group_export_stu2'
+require_relative 'bulk_data_group_export_validation'
+
+module ONCCertificationG10TestKit
+  class MultiPatientAPIGroupSTU2 < Inferno::TestGroup
+    title 'Multi-Patient Authorization and API STU2'
+    short_title 'Multi-Patient API STU2'
+
+    input_instructions %(
+      Register Inferno as a bulk data client with the following information, and
+      enter the client id and client registration in the appropriate fields.
+      This set of tests only checks the Group export. Enter the group export
+      information in the appropriate box.
+
+      Register Inferno with the following JWK Set Url:
+
+      * `#{Inferno::Application[:base_url]}/custom/g10_certification/.well-known/jwks.json`
+    )
+
+    description %(
+      Demonstrate the ability to export clinical data for multiple patients in
+      a group using [FHIR Bulk Data Access
+      IG](https://hl7.org/fhir/uv/bulkdata/). This test uses [Backend Services
```

Comment:
These links should all point to specific IG versions.

Reply:
If you want to update the descriptions of the children, yes, each one needs to be imported and updated indivually.